promscale/visualize-data/grafana.md

```diff
@@ -0,0 +1,105 @@
+# Connect Promscale and Grafana
+Grafana ships with built-in Prometheus, PostgreSQL, and other data source
+plugins that allow you to query and visualize data from a compatible database.
+To add a data source in Grafana you must be signed as a user with organization 
+administration role privileges.
+
+To connect Grafana with Promscale, install and run Grafana version 5.3 or later.
+For information about installing Grafana, see the
+[Grafana installation documentation][grafana-install].
+
+This section shows you how to connect Promscale to Prometheus, Jaeger, and
+PostgreSQL data sources in [Grafana][grafana-homepage].
+
+<procedure>
+
+## Configuring Promscale as Prometheus data source
+
+1.  Navigate to `Configuration` → `Data sources`. The data sources page lists
+    previously configured data sources for the Grafana instance.
+1.  Click `Add data source`to see a list of all supported data sources.
+1.  Type `Prometheus` in the search field and click `Select`.
+1.  Configure the data source:
+    *   In the `Name`field, type `Promscale-metrics`.
+    *   In the `URL` field, type `http://<PROMSCALE-IP-ADDR>:9201`, where
+        `<PROMSCALE-IP-ADDR>` is the IP address of the Promscale instance.
+    *   Use the default values for all other settings.
+
+</procedure>
+
+When you have configured Promscale as a Prometheus data source in Grafana, you
+can create panels that are populated with data using PromQL.
+
+<img class="main-content__illustration"
+src="https://s3.amazonaws.com/assets.timescale.com/images/misc/getting-started-with-promscale-grafana-dashboard.png"
+alt="Sample output for PromQl query"/>
+
+Grafana ships with built-in support for Jaeger, which provides open source,
+end-to-end distributed tracing.
+
+<procedure>
+
+## Configuring Promscale as Jaeger data source
+
+1.  Navigate to `Configuration` → `Data sources`. The data sources page lists
+    previously configured data sources for the Grafana instance.
+1.  Click `Add data source`to see a list of all supported data sources.
+1.  Type `Jaeger` in the search field and click `Select`.
+1.  Configure the data source settings:
+    *   In the `Name`field, type `Promscale-traces`.
+    *   In the `URL`field, type `http://<PROMSCALE-IP-ADDR>:9201`, where
+        `<PROMSCALE-IP-ADDR>`is the IP address of the Promscale instance.
+    *   Use the default values for all other settings.
+
+</procedure>
+
+You can now filter and view traces stored in Promscale using Grafana. To
+visualize your traces, click the **Explore** button to see the traces filtering
```

Comment:
```suggestion
visualize your traces, click the **Explore** button and select the `Promscale-traces` data source to see the traces filtering
```

Replace Promscale-traces with Promscale-Jaeger if you replace the name of the data source above.

Reply:
Hi @ramonguiu ! When I validated the steps I noticed that there was an option to explore at the bottom left when you are configuring the data source. Considering that the user just configured the Jaeger data source and has still not exited the page they may not have to select the `Promscale-traces` ![Screenshot 2022-04-14 at 4 56 19 PM](https://user-images.githubusercontent.com/37059749/163391903-41944d1d-c5d4-4807-9cb2-361a8a422c6d.png)

Reply:
Ah! Sounds great!